NOTES.ANDROID

```diff
@@ -0,0 +1,67 @@
+
+ NOTES FOR THE ANDROID PLATFORMS
+ ===============================
+
+ Requirement details
+ -------------------
+
+ Besides basic tools like perl and make you'll need to download Android
+ NDK. It's avialable for Linux, MacOS X and Windows, but only Linux
+ version was actually tested. There is no reason to believe that MacOS X
+ wouldn't work. And as for Windows it's unclear which "shell" would work
+ [best], MSYS2 might have best chances. NDK version should play lesser
+ role, the goal is to support any recent enough.
+
+ Configuration
+ -------------
+
+ Android is naturally cross-compiled target and you can't use ./config.
+ You have to use ./Configure and name your target explicitly, there are
+ android-arm, android-arm64, android-mips, android-mip64, android-x86
+ and android-x86_64. You might expect you would be required to pass
```

Comment:
I wouldn't make assumptions on what people expect in this kind of documentation...  I suggest starting the sentense with 'Do not pass --cross-compile-prefix or set $CROSS_COMPILE environment variable. as they are "calculated" ...'

Reply:
Well, people are supposed to read *everything*, and in right order, and first-to-read INSTALL says --cross-compile-prefix... I'm just messing with you :-) Thanks for feedback!